APIs/schemas/constraints-schema.json

```diff
@@ -70,6 +70,18 @@
       },
       "rtp_enabled": {
         "$ref": "#/definitions/constraint"
+      },
```

Comment:
I think it might be a good idea to split out constraints relating to particular transport types into different files, as is done with the transport params, then use an anyof. This still works but is maybe a bit more confusing.